tests/test_track_output/results_test.dat

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<VTKFile type="PPolyData" version="0.1" byte_order="LittleEndian" compressor="vtkZLibDataCompressor">
+  <PPolyData GhostLevel="0">
+    <PPoints>
+      <PDataArray type="Float32" Name="Points" NumberOfComponents="3"/>
+    </PPoints>
+    <Piece Source="poly_0.vtp"/>
+  </PPolyData>
+</VTKFile>
```

Comment:
Wow, yeah this has been a recurring issue :-) it's not that I like adding the file, it's just that the test suite doesn't clean it up. I removed it and added this file to .gitignore. Perhaps your or @smharper may want to use a better approach to clean this file up directly in the track output test.